src/training/argument_relation_detection.py

```diff
@@ -113,7 +113,9 @@ def main(
     train_set_json_path = Path(params["train_set_json_path"])
     dev_set_json_path = Path(params["dev_set_json_path"])
     test_set_json_path = params.get("test_set_json_path", None)
-    test_set_output_csv_path = params.get("test_set_output_csv_path", None)
+    test_set_output_csv_path = (
+        Path(params["test_set_output_csv_path"]) if "test_set_output_csv_path" in params else None
+    )

```

Comment:
Holy...I should have writen test_set_output_csv_path by this way
You can merge it.